test/driver.test.js

```diff
@@ -26,6 +26,7 @@ import {
 } from '../bolt-connection/lib/pool/pool-config'
 import { ServerVersion, VERSION_4_0_0 } from '../src/internal/server-version'
 import testUtils from './internal/test-utils'
+import { json as JSON } from 'neo4j-driver-core'
```

Comment:
It's a detail, any specific reason some imports shadow JSON here and change all uses to `json` instead?

Reply:
Sorry, It's a leftover.